config/initializers/sidekiq.rb

```diff
@@ -5,3 +5,6 @@
     chain.add Workers::AuditTrailAttributionMiddleware
   end
 end
+
+require 'sidekiq/web'
```

Comment:
This is already required in routes.rb, maybe we can remove it there.

Reply:
Tried but it errors.